5calls/app/src/main/java/org/a5calls/android/a5calls/view/OutcomeView.java

```diff
@@ -0,0 +1,105 @@
+package org.a5calls.android.a5calls.view;
+
+import android.content.Context;
+import android.content.res.Configuration;
+import android.support.v4.content.ContextCompat;
+import android.support.v7.widget.AppCompatButton;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.LinearLayout;
+
+import org.a5calls.android.a5calls.R;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class OutcomeView extends LinearLayout {
+
+    public static final String VM = "vm";
+    public static final String VOICEMAIL = "voicemail";
+    public static final String CONTACT = "contact";
+    public static final String CONTACTED = "contacted";
+    public static final String UNAVAILABLE = "unavailable";
+
+    private List<Button> buttons;
+
+    public OutcomeView(Context context, String[] outcomes, final Callback callback) {
+        super(context);
+
+        setLayoutParams(new LayoutParams(ViewGroup.LayoutParams.MATCH_PARENT, ViewGroup.LayoutParams.WRAP_CONTENT));
+        setOrientation(VERTICAL);
+
+        if (outcomes != null && outcomes.length > 0) {
```

Comment:
Remove "skip" from outcomes. It doesn't make sense in mobile and @nickoneill has ok'ed this.

Reply:
Done. See: 
https://github.com/5calls/android/pull/89/files?diff=split#diff-d78ff5861ba70301cb399e336d08cdcbR143
https://github.com/5calls/android/pull/89/files?diff=split#diff-d78ff5861ba70301cb399e336d08cdcbR246